cloud/migrate-to-cloud/hypershift.md

```diff
@@ -177,35 +179,45 @@ Use this format:
     accordingly. For example:
 
     ```yml
-    - schema: public
-      name: stocks_real_time
-      time_column_name: time
-      chunk_time_interval: "12h"
-      compress:
-        after: "48h"
-        segmentby:
-          - symbol
-        orderby:
-          - time desc
+    source: "host=<DB_NAME> dbname=<SOURCE_DB_HOSTNAME> user=postgres port=5431"
+    target: "host=<DB_NAME> dbname=<TARGET_DB_HOSTNAME> user=postgres port=5432"
+    verbose: true
+    hypertable_configs:
+      - schema: public
+        name: stocks_real_time
+        time_column_name: time
+        chunk_time_interval: "12h"
+        compress:
+          after: "48h"
+          segmentby:
+            - symbol
+          orderby:
+            - time desc
     ```
 
 1.  At the command prompt, run the Hypershift container. Include the source and
-    destination database connection strings, and the path to your `hypershift.yml`
+    destination database passwords, and the path to your `hypershift.yml`
```

Comment:
@alejandrodnm I don't understand the value of `Include the source and destination database passwords`. The YAML file includes the source and destination DB_URI mentions, I would suggest you to recommend users using the YAML fields to setup DB_URI (along with passwords). This helps to:
1. Reduce the steps to docs. 
2. Doesn't split the config between YAML and CLI options by default. 

Of course, we will add the possibility of configuring DB_URI's in an other file, which contains all details of YAML and CLI options.

Reply:
I think the way the command works has changed since I wrote this. If you can give me the correct command, I'll update the wording appropriately.

Reply:
I think storing passwords in a text file is a bad practice. It might lead to people forgetting to delete them and then they might leak.